storage/src/lib.rs

```diff
@@ -36,23 +36,40 @@ pub fn aleo_dir() -> PathBuf {
     path
 }
 
+///
+/// Returns the workspace path.
+///
+pub fn workspace_dir() -> String {
```

Comment:
maybe we should return the `Path` or `PathBuf` here instead, for greater versatility :thinking:?

Reply:
likely the latter, as `output.stdout()` returns a `Vec<u8>`